bigchaindb/db/utils.py

```diff
@@ -86,33 +85,37 @@ def create_votes_secondary_index(conn, dbname):
     # wait for rethinkdb to finish creating secondary indexes
     r.db(dbname).table('votes').index_wait().run(conn)
 
+def init_database():
+   conn = get_conn()
+   dbname = get_database_name()
+   print(dbname)
+   create_database(conn, dbname)
+
+   table_names = ['bigchain', 'backlog', 'votes']
+   for table_name in table_names:
+     create_table(conn, dbname, table_name)
+
+   create_bigchain_secondary_index(conn, dbname)
+   create_backlog_secondary_index(conn, dbname)
+   create_votes_secondary_index(conn, dbname)
+
 
 def init():
     # Try to access the keypair, throws an exception if it does not exist
     b = bigchaindb.Bigchain()
 
-    conn = get_conn()
-    dbname = get_database_name()
-    create_database(conn, dbname)
-
-    table_names = ['bigchain', 'backlog', 'votes']
-    for table_name in table_names:
-        create_table(conn, dbname, table_name)
-    create_bigchain_secondary_index(conn, dbname)
-    create_backlog_secondary_index(conn, dbname)
-    create_votes_secondary_index(conn, dbname)
+    init_database()
 
     logger.info('Create genesis block.')
     b.create_genesis_block()
     logger.info('Done, have fun!')
 
-
 def drop(assume_yes=False):
     conn = get_conn()
     dbname = bigchaindb.config['database']['name']
-
     if assume_yes:
         response = 'y'
+        print(dbname)
```

Comment:
It was just for seeing the database name, so it should be removed.
Otherwise, yes if it should be maintained, i should use the logger instead of the print.